defaults/main.yml

```diff
@@ -14,6 +14,7 @@ full_clone: true                  # FALSE: linked clone  -  TRUE: full clone ##
 clone_vm: DebianBusterTemplate    # The VM to clone
 clone_storage: local-lvm          # Target storage for full clone.
 clone_format: raw                 # Target drive's backing file's data format. Used only with clone
+clone_target: my_node             # Target node 
```

Comment:
Well, great feature. The detail I see is that the default value for the target variable isn't functional. In case you don't have this variable defined in a place of greater precedence (e.g. host_vars or group_vars), the role will attempt to clone the VM from a template located in the node `my_node`.

I think the best definition for the default value of the variable would be to match the destination node, that is:

```yml
clone_target: "{{ node }}"
```

What do you think?

Reply:
Yes, you are right. 
Thank you.